src/riscv-hybrid-integration.adoc

```diff
@@ -58,23 +58,31 @@ top is 2^MXLEN^
 [#m_bit,reftext="M-bit"]
 === CHERI Execution Mode Encoding
 
-{cheri_default_ext_name} adds a new CHERI execution mode field (M) to
-capabilities within the AP field (see xref:section_cap_perms[xrefstyle=short]):
-
-* When MXLEN=32, the mode is encoded in bit 0 of quadrant 1 from the AP field.
-* When MXLEN=64, the mode is encoded in bit 5 of the AP field: a new <<m_bit>> field is
+{cheri_default_ext_name} adds a new CHERI execution Mode field (M) to
+the capability format, which is only valid for code capabilities, i.e.
+when the <<x_perm>> is set.
+
+* When MXLEN=32, the Mode is encoded in bit 0 of quadrant 1 from the AP field
+_even though it is not a permission_ as shown in <<cap_perms_encoding32>>.
+** Only quadrant 1 represents executable capabilities, and so it's the only
+one which encodes the Mode.
+* When MXLEN=64, the Mode is encoded separately; a new <<m_bit>> field is
 added to the capability format as shown in
 xref:cap_perms_encoding64[xrefstyle=short].
+The <<m_bit>> is only valid for code capabilities, otherwise the field is reserved.
+
+NOTE: Mode is similar to a permission, but is not considered one as it can varied
```

Comment:
> is not considered one as it can varied arbitrarily.

I think of it as orthogonal to permissions entirely. It is a trick to change the behaviour of instructions and the view of CSRs while saving instruction encoding space. I normally see it as an extension bit to the instruction encoding.

Reply:
ok, updated the comment